setup.py

```diff
@@ -111,6 +111,11 @@ def run(self):
     # test_suite = 'nose.collector',
     # test_requires = ['nosetest'],
 
+    tests_require=[
+        # used to provide unit test capabilities
+        'mock >= 2.0.0',
```

Comment:
OK, I'll investigate it. I've added mock because it's convenient to use especially when there isn't any possibility to reach Internet - like during flight.

I'll look at other extractors to see how it's used or use pure Python to mock in worst case scenario.